src/components/Reports/ReportEdit.tsx

```diff
@@ -222,6 +222,33 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     }
   };
 
+  const updateAnnualDetails = (field: string, value: any) => {
+    if (report && report.annualDetails) {
+      setReport(
+        produce((draft) => {
+          // @ts-ignore
+          draft.annualDetails[field] = value;
+        })
+      );
+    }
+  };
+
+  const updateSDGProgress = (index: number, value: string) => {
+    if (
+      report &&
+      report.annualDetails &&
+      report.annualDetails.sustainableDevelopmentGoals &&
+      report.annualDetails.sustainableDevelopmentGoals[index]
```

Comment:
if you prefer.. `report?.annualDetails?.sustainableDevelopmentGoals?[index]`

Reply:
oh.. yeah 😄